### PR TITLE
Initial chart implementation

### PR DESCRIPTION
**Pull request by kopiczko**
Towards https://github.com/giantswarm/roadmap/issues/3229

### CODE REVIEW

File: Makefile.custom.mk [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+HELM_UNITTEST_VERSION := 0.5.0
+HELM_UNITTEST := ./bin/helm-unittest-$(HELM_UNITTEST_VERSION)
+
+.PHONY: gen-schma
+gen-schema: $(HELM_SCHEMA) ## Generates schema.
+	$(HELM_SCHEMA) -l debug
+
+.PHONY: unit-test
+unit-test: $(HELM_UNITTEST) ## Run Helm chart unit tests.
+	$(HELM_UNITTEST) ./helm/ingress-sla-app
```
Comment (kopiczko) [inline Makefile.custom.mk:15]: It would be good to add CI jobs for that, but I'll add it in a separate PR. This one is big enough without it.

**Comment by kopiczko** [comment]
Tested on grizzly WC

**Review by mcharriere — APPROVED** [review]
lgtm!

File: Makefile.custom.mk [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+HELM_SCHEMA_VERSION := 0.11.1
+HELM_SCHEMA := ./bin/helm-schema-$(HELM_SCHEMA_VERSION)
+
+HELM_UNITTEST_VERSION := 0.5.0
+HELM_UNITTEST := ./bin/helm-unittest-$(HELM_UNITTEST_VERSION)
+
+.PHONY: gen-schma
+gen-schema: $(HELM_SCHEMA) ## Generates schema.
+	$(HELM_SCHEMA) -l debug
```
Comment (ubergesundheit) [inline Makefile.custom.mk:11]: I would be careful here. `helm-schema` sets `additionalProperties: false` by default thus blocking possible additional properties. I see you took care of this in your manual schema annotations. And it seems to work how you intended. Just wanted to mention this.

**Review by ubergesundheit — APPROVED** [review]
LGTM

File: Makefile.custom.mk (hunk 2 above)
Comment (kopiczko) [reply Makefile.custom.mk:11]: I tested it on GS WC and it worked fine.

**Pull request merged**